### PR TITLE
EVG-13272 Get aliases in repo if no aliases exist in a project

### PR DESCRIPTION
**Pull request by rayangler**
[[JIRA Ticket](https://jira.mongodb.org/browse/EVG-13272)]

We want to be able to get the aliases of a repo in the event that a project/branch does not have any aliases. We also want to replace instances of `FindAliasInProject` with the new function `FindAliasInProjectOrRepo` where it won't be involved in routes getting/modifying branch/repo settings.

### CODE REVIEW

File: service/project.go [hunk 1]
```diff
@@ -868,7 +868,7 @@ func verifyAliasExists(alias, projectIdentifier string, newAliasDefinitions []mo
 		return true, nil
 	}
 
-	existingAliasDefinitions, err := model.FindAliasInProject(projectIdentifier, alias)
+	existingAliasDefinitions, err := model.FindAliasInProjectOrRepo(projectIdentifier, alias)
```
Comment (malikchaya2) [inline service/project.go:871]: @john-m-liu Is it safe to use this function here since it seems to be connected to the projects page?

**Review by malikchaya2 — APPROVED** [review]
Great work:tada:

File: service/project.go (hunk 1 above)
Comment (john-m-liu) [reply service/project.go:871]: I believe so. This looks like it's now validating that when you change a project, the built-in github PR and tag aliases still exist (if the project is using that functionality), but it's checking the repo as well

**Review by john-m-liu — CHANGES_REQUESTED** [review]
I haven't looked in depth, but there are still 2 places using FindAliasInProject. Did you intentionally skip those?

**Comment by rayangler** [comment]
> I haven't looked in depth, but there are still 2 places using FindAliasInProject. Did you intentionally skip those?

I intentionally skipped those yesterday as I saw that the functions they were being called in would be used in a rest route. Upon another look, it doesn't seem like that rest route actually gets or modifies any branch/repo settings. I'll push out a commit to change them

**Review by john-m-liu — APPROVED** [review]

**Pull request merged**